docs/core/logging-v1.md

```diff
@@ -0,0 +1,808 @@
+---
+title: Logging v1
+description: Core utility
+---
+
+!!! warning
+    Version 1.x.x will continue to be supported until **end of July 2025** for critical bug fixes and security updates in very exceptional cases where you cannot update to v2, but no new features will be added to this version. 
```

Comment:
Our versioniong policy says that we must support versions for at least 6 months. Is that a reasonable time frame? I don't know exactly.

Reply:
I'm not sure I agree with this. I'd rather stick with our versionining policy and keep it for 6 months, especially since we're giving the customer a delayed response since v2 was released and customers will have roughly 1 month to migrate.
What is the impact to keep it for 6 months? Can we keep the pipelines and fix any security issue and stuff?

WDYT?

Reply:
This was decided in another discussion, but I have no issues with keeping 6 months, there is no difference 6 months or 3 months on our side. The main argument is that since there are no breaking changes all users should move to v2. But again no problem with changing, I will update